setup.sh

```diff
@@ -446,6 +447,30 @@ function install_home()
   remove_legacy_git_from_kernel_config_manager
   # Update version based on the current branch
   update_version
+  # Show current environment in terminal
+  show_current_env
+}
+
+function show_current_env()
+{
+  local config_files_path="$etcdir"
+  local config_file_template="${config_files_path}/current_env_content.txt"
+  local shellrc="${HOME}/.bashrc"
+  local content_current_env_file
+
+  if [[ $(ask_yN "Do you want to customize the bash terminal to show the current env?") =~ '1' ]]; then
+    if [[ -f "$config_file_template" ]]; then
+      content_current_env_file="$(< "$config_file_template")"
+    else
+      warning "Setup could not find ${config_file_template}"
+      return 2
```

Comment:
Please, always document the return error. See kw code for more examples.

Reply:
Nice, I'll  document.